test/persistent_receiver_test.go

```diff
@@ -1204,6 +1259,366 @@ var _ = Describe("PersistentReceiver", func() {
 				Expect(receiver.Terminate(10 * time.Second)).ToNot(HaveOccurred())
 			})
 
+
+			for _, autoAck := range []bool{true, false} {
+				autoAckText := ""
+				// ??? I don't speak go, so I'm just rolling with it.
+				autoAck2 := autoAck
```

Comment:
I think you need to a separate scope variable as autoack value changes per declarative iteration.
Instead of using a high scoped variable you could use an entry parameter like the builder function. eg.
```
DescribeTable("Happy cases for outcome configuration on ReceiverBuilder" + autoAckText,
					func(configFunc func(solace.PersistentMessageReceiverBuilder), outcome config.MessageSettlementOutcome)
```
becomes:
```
DescribeTable("Happy cases for outcome configuration on ReceiverBuilder" + autoAckText,
					func(configFunc func(solace.PersistentMessageReceiverBuilder), outcome config.MessageSettlementOutcome, autoAckConfig bool)
```
Then entries have:
```
Entry("Default can Accept", func(builder solace.PersistentMessageReceiverBuilder) {},
						config.PersistentReceiverAcceptedOutcome, 
						autoAck)
```
Though I would suggest something different then autoAck2 for a name. :)

Reply:
Thanks. I have updated this @cjwmorgan-sol

Reply:
Changes look good to me.